rq/group.py

```diff
@@ -68,35 +69,33 @@ def delete_job(self, job_id: str, pipeline: Optional['Pipeline'] = None):
             pipe.execute()
 
     @classmethod
-    def create(cls, connection: Redis, id: Optional[str] = None):
-        return cls(id=id, connection=connection)
+    def create(cls, connection: Redis, name: Optional[str] = None):
+        return cls(name=name, connection=connection)
 
     @classmethod
-    def fetch(cls, id: str, connection: Redis):
+    def fetch(cls, name: str, connection: Redis):
         """Fetch an existing group from Redis"""
-        group = cls(id=id, connection=connection)
-        group.cleanup()
-        if not connection.exists(Group.get_key(group.id)):
+        group = cls(name=name, connection=connection)
+        if not connection.exists(Group.get_key(group.name)):
             raise NoSuchGroupError
         return group
 
     @classmethod
-    def cleanup_group(cls, id: str, connection: Redis, pipeline: Optional['Pipeline'] = None):
+    def cleanup_group(cls, name: str, connection: Redis, pipeline: Optional['Pipeline'] = None):
```

Comment:
Should we make this into an instance method? `group.cleanup()`

Reply:
Done